### ISSUE TITLE
Goldilocks 6.1.0 chart SHA changed

### ISSUE THREAD

**Issue by grzechukol** [reactiveops/charts#849]
Hi, I want to report that `https://charts.fairwinds.com/stable/goldilocks-6.1.0.tgz` SHA256 changed from `2c11ed03e18a1103168685aff04fc66138e37ac941c3ce95e05d311bff20470d` to `64edd619ff50d727910f0c453c23e2af4450495f45b1a2da94e683a5953874ad`.

Please let me know if it was an intended change or if something happened.

**Comment by sudermanjr** [comment]
The way our automation works, we sync all the charts every time we merge one, so it is not entirely unexpected that a sha could change at some point.

**Comment by grzechukol** [comment]
For some users, that could be problematic as checking SHAs of externally downloadable artefacts is a good practice. Is there anything you could do to make those builds deterministic?

**Comment by sudermanjr** [comment]
Agreed, we don't love that it works this way right now. We're having an internal conversation about the way we want to resolve this, but we likely won't have a quick solution

**Comment by grzechukol** [comment]
@sudermanjr Shouldn't this ticket be reopenned?

**Comment by sudermanjr** [comment]
Yeah, thanks. I've pinned it so it won't go stale

**Comment by jasaltvik** [comment]
@sudermanjr I was trying to figure out why the ArtifactHub RSS feed keeps telling me there's a new chart update for Golidlocks, even though it's the same version over and over again. I assume this is the same underlying issue related to your automation? 

<img width="697" alt="bilde" src="https://github.com/user-attachments/assets/05b6f804-e957-43c1-943b-f02a6a2ba88c">

**Comment by sudermanjr** [comment]
We fixed this recently by changing the sync script to only sync the latest charts
